cloud/deploy/deploy_test.go

```diff
@@ -137,7 +136,6 @@ var (
 			IsDagDeployEnabled:       true,
 			IsCicdEnforced:           false,
 			Type:                     &hybridType,
-			DagTarballVersion:        &tarballVersion,
```

Comment:
Why are we removing this?

Reply:
It is no longer needed for the test. DesiredDagTarballVersion is used instead